src/web/components/ClientSideTokenGeneration/CstgDomainsTable.scss

```diff
@@ -0,0 +1,59 @@
+.cstg-domains-management {
+  .cstg-domains-table {
+    width: 100%;
+    th:first-of-type,
+    td:first-of-type {
+      padding-left: 10px;
+      width: 21px;
+    }
+    th.action {
+      text-align: end;
+      padding-left: 140px;
+    }
+    tr {
+      .tristate-checkbox {
+        margin-right: 16px;
+      }
+
+      button[role='checkbox'] {
+        width: 21px;
+        height: 21px;
+      }
+      .input-container {
+        height: 26px;
+      }
+    }
+
+    .domain {
+      min-width: 220px;
```

Comment:
I'm always cautious of using larger pixel values like this (and also the `th.action { padding-left: 140px; }` above). Could you achieve the effect you're after by using flex-grow?

Reply:
I think the table does not support flex-grow. We used to use percentages to control the width of the column. 

In this case, what I want to achieve is to make sure the domain column by default is wider than the text input. Maybe give the action column  a percentage is a better approach to set min-width?

Reply:
I've updated it to percentage width and remove the padding-left as I found in our case we don't need that padding